src/Symfony/Bundle/FrameworkBundle/Templating/TemplateReference.php

```diff
@@ -43,8 +43,15 @@ public function getPath()
         $controller = str_replace('\\', '/', $this->get('controller'));
 
         $path = (empty($controller) ? '' : $controller.'/').$this->get('name').'.'.$this->get('format').'.'.$this->get('engine');
+        if (empty($this->parameters['bundle'])) {
+            if (file_exists('views/')) {
```

Comment:
this looks fragile to me: we don't know where PHP will look for the file (include_path, etc?)

Reply:
How could we do it ?
I guess we need to check the existance or adding a parameters ?